### PR TITLE
added toggle

### PR DESCRIPTION
**Pull request by Mare0804**
# *Name of the PR*

## :recycle: Current situation
*Describe the current situation (if possible with and exemplary (or real) code snippet and/or where this is used)*

## :bulb: Proposed solution
*Describe the solution (if possible with and exemplary (or real) code snippet)*

### Problem that is solved
*Provide a description and link issues that are solved*

### Implications
*Describe the implications, e.g. refactoring*

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested and which corner cases are missing?*

### Reviewer Nudging
*Where should the reviewer start, where is a good entry point?*

### CODE REVIEW

File: frontend/src/features/zoomableChart/zoomableChart.js [hunk 1]
```diff
@@ -63,24 +64,7 @@ export const ZoomableChart = () => {
...
-          {
-            icon: '<img src="TDS.png" width="20">',
-            index: 7,
-            title: 'TDS',
-            class: 'custom-icon',
-            click: (chart, options, e) => {
-              console.log("clicked custom-icon")
-            }
-          },
+          customIcons: [
```
Comment (ChrGrb) [inline frontend/src/features/zoomableChart/zoomableChart.js]: Do we need this?

File: frontend/src/features/zoomableChart/zoomableChart.js [hunk 2]
```diff
@@ -141,7 +125,7 @@ export const ZoomableChart = () => {
     }
   }, [data]);
   return (
-    <div id="chart">
+    <div id="chart" >
```
Comment (ChrGrb) [inline frontend/src/features/zoomableChart/zoomableChart.js]: Why?

File: frontend/src/features/zoomableChart/zoomableChart.js [hunk 3]
```diff
@@ -17,6 +17,7 @@ export const ZoomableChart = () => {
   const dataState = useSelector((state) => state.data.dataBySpanState);
 
   const [selectedSpan, setSelectedSpan] = useState("fiveYears");
+  const [selectedData, setSelectedData] = useState("pH")
```
Comment (ChrGrb) [inline frontend/src/features/zoomableChart/zoomableChart.js]: Vewwy pwetty

**Review by ChrGrb — APPROVED** [review]
